testcases/TC06.1-1-fail-1.md

```diff
@@ -0,0 +1,55 @@
+---
```

Comment:
the page title for this page is generic. Is the page title coming from the front matter?

Reply:
Ah, yes, good catch! The reference in the meta.html include incorrectly referenced {{ site.title }} instead of {{ page.title }}. Just fixed it.